cmd/gitsink/v1/gitsink.go

```diff
@@ -56,61 +56,98 @@ func Execute() { // hello
 	p := kingpin.MustParse(app.Parse(os.Args[1:]))
 
 	config := config.Parse()
-	fmt.Printf("%+v\n", config)
 
 	switch p {
 
 	case appSync.FullCommand():
-		log.Infof("SYNC")
-		log.Infof("App Log Level: %v\n", *appLogLevel)
-		log.Infof("Run Once: %v\n", *appSyncRunOnce)
+		fmt.Println("SYNC")
```

Comment:
Curious why initial statements are using fmt instead of logger.

Reply:
Just kept them as placeholders, will update.